src/NuGet.Clients/NuGet.Indexing/NuGetQuery.cs

```diff
@@ -40,50 +40,52 @@ public static Query MakeQuery(string q)
 
         static Query ConstructQuery(Dictionary<string, HashSet<string>> clauses)
         {
-            Analyzer analyzer = new PackageAnalyzer();
-
             BooleanQuery query = new BooleanQuery();
-            foreach (var clause in clauses)
+
+            using (var analyzer = new PackageAnalyzer())
```

Comment:
can we enable C# 8 in some of these so we can just write:

>   using PackageAnalyzer analyzer = new PackageAnalyzer();

And avoid all the formatting changes. 
I think we have C# 8 in some other projects such as NuGet.PackageManagement.UI.

Reply:
I like the idea. As per [doc](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version#defaults), all .NET Framework projects default to C# 7.3 and `Choosing a language version newer than the default can cause hard to diagnose compile-time and runtime errors. `

I will update the language version to 8 for projects modified in this PR and see if I encounter any new errors.

Reply:
Pushed a new commit yesterday and has a green CI build.